feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/domain/nominationPools/selecting/SelectingNominationPoolInteractor.kt

```diff
@@ -1,19 +1,22 @@
-package io.novafoundation.nova.feature_staking_impl.domain.staking.pool
+package io.novafoundation.nova.feature_staking_impl.domain.nominationPools.selecting
 
 import io.novafoundation.nova.feature_staking_impl.data.StakingOption
+import io.novafoundation.nova.feature_staking_impl.data.chain
+import io.novafoundation.nova.feature_staking_impl.data.nominationPools.pool.KnownNovaPools
+import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.common.getPoolComparator
 import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.model.NominationPool
 import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.model.apy
 import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.pools.NominationPoolProvider
 import kotlinx.coroutines.CoroutineScope
 
 class SelectingNominationPoolInteractor(
-    private val nominationPoolProvider: NominationPoolProvider
+    private val nominationPoolProvider: NominationPoolProvider,
+    private val knownNovaPools: KnownNovaPools
 ) {
 
     suspend fun getSortedNominationPools(stakingOption: StakingOption, coroutineScope: CoroutineScope): List<NominationPool> {
         val nominationPools = nominationPoolProvider.getNominationPools(stakingOption, coroutineScope)
-        val comparator = compareByDescending<NominationPool> { it.apy }
-            .thenByDescending { it.membersCount }
+        val comparator = getPoolComparator(knownNovaPools, stakingOption.chain)
         return nominationPools.sortedWith(comparator)
```

Comment:
As we discussed yesterday, we still need to apply same filterting in pool list screen that we use for pool recommendations. On the other hand, pool search wont have any filters

Reply:
Yes, Agree with you. I already made it in search pools screen. I hope it's okay since this screen also use this interactor